bdb/file.c

```diff
@@ -7084,6 +7084,19 @@ static size_t dirent_buf_size(const char *dir)
                                              : sizeof(struct dirent));
 }
 
+uint64_t bdb_queuedb_size(bdb_state_type *bdb_state) 
+{
+    char tmpname[PATH_MAX];
+    struct stat st;
+    snprintf(tmpname, sizeof(tmpname), "%s/%s.queuedb", bdb_state->dir, bdb_state->name);
```

Comment:
Can we make this snprintf a generic function get_queue_filename() and use it from all the places that get the filename?

Reply:
This is the only such place, unless you count form_queuedb_name, which has a different set of rules (it forms a loggable name with our usual XXX. kludge).  It can be moved to a different function, sure.

Reply:
If it is the only place, then leave it : )